numba_dpex/experimental/core/types/kernel_api/items.py

```diff
@@ -7,6 +7,31 @@
 from numba.core import errors, types
 
 
+class GroupType(types.Type):
+    """Numba-dpex type corresponding to :class:`numba_dpex.kernel_api.Group`"""
+
+    def __init__(self, ndim: int):
```

Comment:
Constructor does not seem right.

Reply:
This is a type constructor:
https://developer.codeplay.com/products/computecpp/ce/2.11.0/api-reference/classcl_1_1sycl_1_1group